src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -287,16 +304,23 @@ export default {
             }
 
             this.persist(domainObjectStyles);
+            if (this.stopProvidingTelemetry) {
+                this.stopProvidingTelemetry();
+            }
         },
         updateDomainObjectItemStyles(newItems) {
             //check that all items that have been styles still exist. Otherwise delete those styles
             let domainObjectStyles =  (this.domainObject.configuration && this.domainObject.configuration.objectStyles) || {};
             let itemsToRemove = [];
             let keys = Object.keys(domainObjectStyles);
+            //TODO: Need an easier way to find which properties are itemIds
```

Comment:
I think the difficulty here is arising from `domainObject.configuration.objectStyles` being used to store more than just styles. `defaultConditionId`, `selectedConditionId`, `conditionSetIdentifier` should be stored elsewhere. Out of scope for this change, perhaps a followup?

Reply:
I don't think they should be stored elsewhere as these properties are only relevant if objectStyles (static or conditional) have been applied to a domainObject.